packages/buendia-dashboard/data/usr/share/buendia/dashboard/index

```diff
@@ -17,6 +17,17 @@ tail -5 /var/log/buendia/buendia-warmup.log | grep -q Success && updown=SERVING
 
 uptime=$(uptime | sed -e 's/.* up//' -e 's/, *[0-9][0-9]* *user.*//')
 
+up_count=$(cat /var/log/buendia/buendia-warmup.log | grep '(end, status 0)' | wc -l)
```

Comment:
Please refactor these into a common script / location so that if we need to modify the logic we only need to modify it in one place.